datafiles/templates/Html/package-page.html.st

```diff
@@ -89,6 +89,13 @@
           <td>$package.maintainer$</td>
         </tr>
 
+        $if(hackage.hasUpdateTime)$
+        <tr>
+          <th>Revisions</th>
```

Comment:
"Revisions" sounds odd to me; iirc I suggested somewhere on that long thread to use "Revised" instead (as it aligns better with the other label "Uploaded")

Reply:
sure, changed.